runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -1334,57 +1329,6 @@ public void testTransformTranslator() throws IOException {
     assertTrue(transform.translated);
   }
 
-  @Test
-  public void testSdkHarnessConfiguration() throws IOException {
```

Comment:
Any reason that we want to remove this test case?

Reply:
This test seems to check whether the environment id and docker url are populated correctly in beam fnapi proto. I think we want to keep this test. But we can remove `use_runner_v2 ` from the test setup because of the dual job submission.